src/JavierLeon9966/Shield/Shield.php

```diff
@@ -30,7 +30,7 @@ final class Shield extends PluginBase implements Listener{
 	private array $cooldowns = [];
 
 	public function onEnable(): void{
-		$shield = new ShieldItem(new ItemIdentifier(ItemIds::SHIELD, 0), 'Shield');
+		$shield = new ShieldItem(new ItemIdentifier(ItemTypeIds::FIRST_UNUSED_ITEM_ID), 'Shield');
```

Comment:
This is not what I meant, here’s what I meant:
https://github.com/jasonwynn10/NativeDimensions/blob/07c357255560941fe644b6648db8b78bbeb99757/src/Main.php#L101-L108